/**:
  ros__parameters:
    num_previous_frames: 4
    x_min: 10.0  # meters
    x_max: 40.0 # meters
    y_min: -20.0 # meters
    y_max: 20.0  # meters
    grid_resolution: 0.3 # meters